.github/workflows/build.yml

```
@@ -0,0 +1,42 @@
+name: build
+on:
+  # Enable manually triggering this workflow via the API or web UI
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.19.x
```

Why an old Go version?
```suggestion
          go-version: 1.20.x
```

---

I'm keeping it aligned with what's in k6's `go.mod`.

---

The go mod's version defines the minimum _supported_ version, not the version that [we use for building](https://github.com/grafana/k6/blob/5b5c5ccbb869d45a962b22b2f8f343cc0a01b906/.github/workflows/build.yml#L29).

---

Fair enough, I wanted to be on the safe side, will switch 👍🏻